flask_iiif/restful.py

```diff
@@ -75,7 +76,10 @@ def get(self, version, uuid):
             width, height = map(int, cached.split(','))
         else:
             data = current_iiif.uuid_to_image_opener(uuid)
-            image = IIIFImageAPIWrapper.open_image(data)
+            try:
+                image = IIIFImageAPIWrapper.open_image(data)
+            except:
```

Comment:
```suggestion
            except Exception:
```

In general `try/except` without a specified exception type [is a bad practice](https://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice). 

Besides that though, I'm also a bit worried about adding a catch-all here, since e.g. some of these are already handled in [`IIIFImageAPIWrapper.open_image`](https://github.com/inveniosoftware/flask-iiif/blob/80dd9ce28bec2334bdefdf3c246c67b1408a4203/flask_iiif/api.py#L581-L585). If you came across other types of exceptions, it's better to explicitly catch and handle them there, otherwise a 500 response might actually be fine (i.e. the server screwed-up and we're not sure what happened, this is also useful for logs).

Reply:
That's a good tip, thanks.

Reply:
On closer inspection,`IIIFImageAPIWrapper.open_image` does already cause a 404 as it should. invenio-iiif is causing the 500 error - I will revert this commit / close #73 and raise an issue there.